src/bin/facilitator-init.ts

```diff
@@ -30,8 +31,63 @@ import Repositories from '../repositories/Repositories';
 import SeedData from '../SeedData';
 import GatewayAddresses from '../Config/GatewayAddresses';
 
+/**
+ * It provides gateway addresses and origin chain id. It is to be used when mosaic
+ * config is provided.
+ * @param auxChainId Auxiliary chain id.
+ * @param mosaicConfigPath Path to mosaic config.
+ * @returns originchain id and gatewayaddresses object.
+ */
+function getFromMosaicConfig(
```

Comment:
Please move these methods to some other class.

Reply:
You could move whole logic to file `lib/facilitator-init` and commander method will have only one statement of calling  class in lib.

Reply:
I would suggest to create new ticket for moving other logic to file **lib/facilitator-init**

Reply:
Created ticket https://github.com/mosaicdao/facilitator/issues/240 to address it.